File: Ontime/Frontend/App/app/src/main/java/com/dytstudio/signup/Models/Address.java
package com.dytstudio.signup.Models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Mo on 28/11/2017.
 */

public class Address {

    @SerializedName("addressID")
    @Expose
    public Integer addressID;
    @SerializedName("streetname")
    @Expose
    public String streetname;
    @SerializedName("housenumber")
    @Expose
    public String housenumber;
    @SerializedName("city")
    @Expose
    public String city;
    @SerializedName("zipcode")
    @Expose
    public String zipcode;
    @SerializedName("country")
    @Expose
    public String country;

    public Address(Integer addressID, String streetname, String housenumber, String city, String zipcode, String country) {
        this.addressID = addressID;
        this.streetname = streetname;
        this.housenumber = housenumber;
        this.city = city;
        this.zipcode = zipcode;
        this.country = country;
    }

}

File: Ontime/Backend/OnTimeBackend/Data/DatabaseModels/Address.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Uber4Cream.Data.DatabaseModels
{
    public class Address
    {
        public int AddressID { get; set; }
        public string streetname { get; set; }
        public string housenumber { get; set; }
        public string city { get; set; }
        public string zipcode { get; set; }
        public string Country { get; set; }

        public Employer employer { get; set; }

       public string GetAddress()
        {

            return streetname + "+" + housenumber + "+" + city + "+" + zipcode + "+" + Country;
        }  

    }
}
